addons/pharma/functions/fnc_getBloodVolumeChange.sqf

```diff
@@ -42,17 +43,26 @@ if (!isNil {_unit getVariable [QACEGVAR(medical,ivBags),[]]}) then {
     private _incomingVolumeChange = [0,0,0,0,0,0];
     private _fluidWarmer = _unit getVariable [QEGVAR(hypothermia,fluidWarmer), [0,0,0,0,0,0]];
     private _fluidHeat = 0;
-
+    private _incomingFlowAmount = [0,0,0,0,0,0];
+    
     _bloodBags = _bloodBags apply {
         _x params ["_bagVolumeRemaining", "_type", "_bodyPart"];
 
         private _tourniquets = GET_TOURNIQUETS(_unit);
 
         if ((_tourniquets select _bodyPart isEqualTo 0) && (_IVarray select _bodyPart isNotEqualTo 3)) then {
             private _IVflow = _unit getVariable [QGVAR(IVflow), [0,0,0,0,0,0]];
-            private _bagChange = (_flowCalculation * (_IVflow select _bodyPart)) min _bagVolumeRemaining; // absolute value of the change in miliLiters
+            private _viscosity = 1
+            switch (true) do {
```

Comment:
Don't make this a case statement, store fluid viscosity as part of the fluid class in ACE_Medical_Treatment